zcash_client_backend/src/fees.rs

```diff
@@ -488,7 +488,7 @@ impl EphemeralBalance {
 /// by a transaction having a specified set of inputs and outputs.
 pub trait ChangeStrategy {
     type FeeRule: FeeRule + Clone;
-    type Error;
+    type Error: From<<Self::FeeRule as FeeRule>::Error>;
```

Comment:
This change needs to be noted in the CHANGELOG

Reply:
I don't recall doing this change to be honest. Maybe clippy  or compiler suggestion? I'll dig in

Reply:
This was one of the things we did in our pairing; it just needs to be noted in the CHANGELOG